Settings.go

```diff
@@ -29,16 +29,17 @@ const (
 
 // Settings represents user settings.
 type Settings struct {
-	UserID        string               `json:"userId"`
-	SortBy        string               `json:"sortBy"`
-	TitleLanguage string               `json:"titleLanguage" editable:"true"`
-	Providers     ServiceProviders     `json:"providers"`
-	Avatar        AvatarSettings       `json:"avatar"`
-	Format        FormatSettings       `json:"format"`
-	Notification  NotificationSettings `json:"notification"`
-	Editor        EditorSettings       `json:"editor"`
-	Privacy       PrivacySettings      `json:"privacy"`
-	Theme         string               `json:"theme" editable:"true"`
+	UserID           string               `json:"userId"`
+	SortBy           string               `json:"sortBy"`
+	TitleLanguage    string               `json:"titleLanguage" editable:"true"`
+	Providers        ServiceProviders     `json:"providers"`
+	Avatar           AvatarSettings       `json:"avatar"`
+	Format           FormatSettings       `json:"format"`
+	Notification     NotificationSettings `json:"notification"`
+	Editor           EditorSettings       `json:"editor"`
+	Privacy          PrivacySettings      `json:"privacy"`
+	Theme            string               `json:"theme" editable:"true"`
+	CalendarSettings CalendarSettings     `json:"calendar" editable:"true"`
```

Comment:
Similar to the `Notification` field of type `NotificationSettings`, this should be the `Calendar` field of type `CalendarSettings`.
The `Settings` suffix should be removed as the struct name already implies it.